guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,51 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+* If you want both servers to manage content synchronization and content view creation, follow these guidelines to prevent content drift:
+** Regularly synchronize repositories on both servers.
+You can use the following Ansible modules to automate repository synchronization: `{ansible-namespace}.repository_sync` and `{ansible-namespace}.sync_plan`.
+** Ensure that content views on both servers match.
+* If you want one server to manage content synchronization and content view creation, use one of these features to prevent content drift:
+** If your disaster recovery site has network access to your primary site, use Inter-Server Synchronization (ISS) to ensure your disaster recovery server synchronizes its content from the primary server.
+** If your disaster recovery site does not have network access to your primary site, synchronize content by using export and import.
+* If you want one server to manage only content view creation but not content synchronization, you can configure the other server or multiple other servers to import content views from the first server but synchronize content from repositories.
+//      |------------ Satellite for CVs only ---------------|
+//                /                               \
+//              /                                   \
+//            /                                       \
+// |--Satellite for Hosts --|     |--Satellite for hosts--|
+. Register hosts to your servers so that each server manages hosts in its respective data center.
```

Comment:
This is ambigous in my eyes. Users should register all hosts in data center 1 to Foreman/Katello in DC1; and all hosts in DC2 to Foreman/Katello in DC2.

AFAIK there is no other way than doing it like this unless you have two Foreman/Katello or Smart Proxy Servers that manage hosts in the same network. I don't know if that's a good idea.

@ehelms Can you please ACK line 32 or do you suggest to reword it in some way?

Reply:
That re-wording you suggest is more specific, and I think we can go that way. I tend to like "should" language in this scenario to denote best practice but not hard requirement.

Reply:
The word "should" might trigger peer reviewers or, worse, the Vale linter. How about we add the more specific wording as an example that extends the current line 32? Does the latest version work for you @maximiliankolb @ehelms?

Reply:
I like line 32+33 :+1: